reference/7.2/Microsoft.PowerShell.Utility/Out-File.md

```diff
@@ -133,6 +133,26 @@ provider `Alias:`. The `Get-Location` cmdlet displays the complete path for `Ali
 **C:\TestDir\AliasNames.txt**. The `Get-Content` cmdlet uses the **Path** parameter and displays the
 file's content in the PowerShell console.
 
+### Example 5: Set file output width for entire script
+
+This example uses `$PSDefaultParameterValues` to set the `Width` parameter for all invocations of
+`Out-File` and the redirection operartors (`>` and `>>`) to 2000.  This is an easy way to ensure
+that everywhere in a script you output table formatted data to file, PowerShell will use a line
+width of 2000 instead of a line width determined by the PowerShell host's console width.
+
+```powershell
+param()
+$PSDefaultParameterValues['out-file:width'] = 2000
+
+$logFile = [System.IO.Path]::ChangeExtension($PSCommandPath, ".log")
+
+Get-ChildItem Env:\ > $logFile
+
+Get-Service -ErrorAction Ignore | Format-Table -AutoSize | Out-File $logFile -Append
+
+Get-Process | Format-Table Id,SI,Name,Path,MainWindowTitle >> $logFile
```

Comment:
I suggest making this into a function. As written, it may not be clear that this is a script. Also, if someone copy/paste this to the command line then the preference becomes global to that session. Wrapping it in a function is safer for the reader.

Reply:
Done.